### ISSUE TITLE
Non-Functional Genghis.php

### ISSUE THREAD

**Issue by NoiSek** [bobthecow/genghis#11]
I thought it was just because I'd left out the src and assets folder from the git repo, but even after adding them this is the only thing on the screen: http://puu.sh/x5TW

I'd thought that a majority of the functionality and css/js assets were packed into genghis.php, but as this screen shows it isn't exactly giving stellar results.

NGINX + PHP5-FPM + Mongo 1.2.10

**Comment by bobthecow** [comment]
For nginx, you'll need a config similar to the one [in this comment](https://github.com/bobthecow/genghis/issues/6#issuecomment-3724632). Add that and let me know if it works for you?

**Comment by NoiSek** [comment]
Apologies for the late reply, thought I'd already done so. No dice, unfortunately. I've set up php5-fpm to listen on that socket, but I still only see this text and nothing else:

```
Genghis

Genghis, by Justin Hileman.

Keyboard shortcuts available
```

**Comment by bobthecow** [comment]
@jmikola helped me track down a related issue, so this should be fixed in v2.0. Feel free to reopen if the latest release didn't fix it for you.
